lib/bundled_gems.rb

```diff
@@ -101,8 +101,9 @@ def self.find_gem(path)
   def self.warning?(name, specs: nil)
     # name can be a feature name or a file path with String or Pathname
     feature = File.path(name)
-    # bootsnap expand `require "csv"` to `require "#{LIBDIR}/csv.rb"`
-    name = feature.delete_prefix(LIBDIR).chomp(".rb").tr("/", "-")
+    # bootsnap expands `require "csv"` to `require "#{LIBDIR}/csv.rb"`,
+    # and `require "syslog"` to `require "#{ARCHDIR}/syslog.so"`.
+    name = feature.delete_prefix(ARCHDIR).delete_prefix(LIBDIR).chomp(".rb").tr("/", "-")
```

Comment:
can the latter methods on this chain be changed to the `!` varieties, so we only make one new string, rather than 4?

Reply:
Sure, I pushed an update.

In the process I realised that `.chomp(".rb")` is redundant with `.sub!(LIBEXT, "")`, so I removed it.